src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+/* stylelint-disable rule-empty-line-before */
+/* stylelint-disable no-descending-specificity */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(/src/images/star-active.svg);
+}
```

Comment:
The task requires each BEM block to have its own separate file. Currently, all styles are in a single CSS file. Please adjust the file structure to meet the task's core requirements.